client/blocks/express-checkout/index.js

```diff
@@ -1,20 +1,44 @@
 import { PAYMENT_METHOD_EXPRESS_CHECKOUT_ELEMENT } from './constants';
-import { ExpressCheckout } from './express-checkout';
+import { ExpressCheckoutContainer } from './express-checkout-container';
 import ApplePayPreview from './apple-pay-preview';
+import GooglePayPreview from './google-pay-preview';
 import { loadStripe } from 'wcstripe/blocks/load-stripe';
 import { getBlocksConfiguration } from 'wcstripe/blocks/utils';
 
 const stripePromise = loadStripe();
 
-const expressCheckoutElementsPaymentMethod = {
-	name: PAYMENT_METHOD_EXPRESS_CHECKOUT_ELEMENT,
-	content: <ExpressCheckout stripe={ stripePromise } />,
+const expressCheckoutElementsGooglePay = ( api ) => ( {
+	name: PAYMENT_METHOD_EXPRESS_CHECKOUT_ELEMENT + '_googlePay',
+	content: (
+		<ExpressCheckoutContainer
+			api={ api }
+			stripe={ stripePromise }
+			expressPaymentMethod="googlePay"
+		/>
+	),
+	edit: <GooglePayPreview />,
+	canMakePayment: () => true,
+	paymentMethodId: PAYMENT_METHOD_EXPRESS_CHECKOUT_ELEMENT,
+	supports: {
+		features: getBlocksConfiguration()?.supports ?? [],
+	},
+} );
+
+const expressCheckoutElementsApplePay = ( api ) => ( {
+	name: PAYMENT_METHOD_EXPRESS_CHECKOUT_ELEMENT + '_applePay',
+	content: (
+		<ExpressCheckoutContainer
+			api={ api }
+			stripe={ stripePromise }
+			expressPaymentMethod="applelePay"
```

Comment:
```suggestion
			expressPaymentMethod="applePay"
```

Typo. :)

Reply:
Thanks @james-allan 🤣 
Fixed in [7a21842](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3439/commits/7a218427e8fbec12cff6ee8f7732759cbd701913)